src/frontend/services/api.service.ts

```diff
@@ -0,0 +1,57 @@
+import {Injectable} from '@angular/core';
```

Comment:
Passt für mich, habe bereits in meinem Branch folgendes gemacht
https://github.com/mtinner/CAS-FEE_Project2/blob/feature/addCardComponent/src/frontend/components/pages/shopping-list.service.ts